sema/commons/yml/loader.py

```
@@ -0,0 +1,42 @@
+from yaml import SafeLoader, Loader
+from yaml.nodes import ScalarNode
+from logging import getLogger, Logger
+
+
+log: Logger = getLogger(__name__)
+
+
+class LoaderBuilder():
```

Ruff (0.8.2)

Unnecessary parentheses after class definition

Remove parentheses

(UP039)